codersdk/workspaceagents.go

```diff
@@ -53,7 +54,9 @@ type WorkspaceAgent struct {
 	Version              string               `json:"version"`
 	Apps                 []WorkspaceApp       `json:"apps"`
 	// DERPLatency is mapped by region name (e.g. "New York City", "Seattle").
-	DERPLatency map[string]DERPRegion `json:"latency,omitempty"`
+	DERPLatency              map[string]DERPRegion `json:"latency,omitempty"`
+	ConnectionTimeoutSeconds int32                 `json:"connection_timeout_seconds"`
```

Comment:
@BrunoQuaresma Is this a preferable format, or does a duration string work as well for the frontend (e.g. `2m0s`).

**NOTE:** This is not a field that the frontend needs to care about unless it's useful, the `status` changes automatically.